docs/victory-theme/ecology.md

```diff
@@ -0,0 +1,311 @@
+
+Using VictoryTheme.material for a component's ```theme``` prop applies a clean, bright style to your charts.
+
+VictoryTheme works across all component types - it can be applied to chart components themselves.
+
+``` playground
+<svg>
+  <VictoryAxis
+    theme={VictoryTheme.material}
+  />
+  <VictoryCandlestick
+    theme={VictoryTheme.material}
+  />
+</svg>
+```
+
+It can be applied to VictoryStack or VictoryGroup, and then passed down to wrapped chart components.
+
+```playground
+<VictoryStack theme={VictoryTheme.material}>
+  <VictoryArea/>
+  <VictoryArea/>
+  <VictoryArea/>
+  <VictoryArea/>
+  <VictoryArea/>
+  <VictoryArea/>
+</VictoryStack>
+```
+
+It can even be applied to VictoryChart and passed down to all subsequent child components.
+
+```playground
+<VictoryChart theme={VictoryTheme.material}>
+  <VictoryGroup
+    height={500}
+    offset={20}
+  >
+    <VictoryBar
+      data={[
+        {x: 1, y: 1},
+        {x: 2, y: 2},
+        {x: 3, y: 3}
+      ]}
+    />
+    <VictoryBar
+      data={[
+        {x: 1, y: 2},
+        {x: 2, y: 1},
+        {x: 3, y: 1}
+      ]}
+    />
+    <VictoryBar
+      data={[
+        {x: 1, y: 3},
+        {x: 2, y: 4},
+        {x: 3, y: 2}
+      ]}
+    />
+  </VictoryGroup>
+</VictoryChart>
+```
+
+### Create Your Own Custom Theme Object
+
+Most chart types only need styles from a theme object, like VictoryScatter and VictoryLine, below.
+
+```playground_norender
+class App extends React.Component {
+  render() {
+    const myTheme = {
+      scatter: {
+        data: {
+          fill: "red"
+        },
+        labels: {
+          padding: -25,
+          fontFamily: "Garamond"
+        },
+        // even if one of the subobjects isn't being utilized,
+        // it needs to exist for the theme to be supported
+        parent: {}
+      },
+      line: {
+        data: {
+          stroke: "red",
+          fill: "none"
+        },
+        labels: {
+          padding: 10,
+          fontFamily: "Garamond"
+        },
+        parent: {}
+      }
+    };
+
+    const data = [
+      {x: 1, y: 1},
+      {x: 2, y: 4},
+      {x: 3, y: 2},
+      {x: 4, y: 6},
+      {x: 5, y: 5},
+      {x: 6, y: 9}
+    ];
+
+    return (
+      <svg>
+        <VictoryScatter
+          data={data}
+          size={7}
+          theme={myTheme}
+          labels={["one", "two", "three", "four", "five", "six"]}
+        />
+        <VictoryLine
+          data={data}
+          theme={myTheme}
+          label="Hello"
+        />
+      </svg>
+    )
+  }
+}
+
+ReactDOM.render(<App/>, mountNode);
+```
+
+Other chart types, like VictoryCandlestick and VictoryPie, require that props be a part of their theme object.
+
+```playground_norender
+class App extends React.Component {
+  render() {
+    const myTheme = {
+      candlestick: {
+        style: {
+          data: {
+            stroke: "none"
+          },
+          labels: {},
+          parent: {}
+        },  
+        props: {
```

Comment:
@boygirl hmm I guess more along the lines of like if they wanted to add candleColors or whatever to a candlestick chart in the theme, which goes through props. there might be clearer wording for that though, so I'll work on that